broadcaster/_backends/kafka.py

```diff
@@ -0,0 +1,49 @@
+import typing
+from urllib.parse import urlparse
+from .base import BroadcastBackend
+from .._base import Event
+from aiokafka import AIOKafkaProducer, AIOKafkaConsumer
+import asyncio
+
+
+class KafkaBackend(BroadcastBackend):
+    def __init__(self, urls: typing.Union[str, typing.List[str]]):
+        if isinstance(urls, str):
+            urls = [urls]
+
+        # Kafka accept n + 1 servers to bootstrap initial cluster metadata.
+        # https://aiokafka.readthedocs.io/en/stable/api.html#aiokafkaconsumer-class
+        self._servers = [urlparse(url).netloc for url in urls]
+        self._consumer_channels: typing.Set = set()
+
+    async def connect(self) -> None:
+        loop = asyncio.get_event_loop()
+        self._producer = AIOKafkaProducer(loop=loop, bootstrap_servers=self._servers)
+        self._consumer = AIOKafkaConsumer(loop=loop, bootstrap_servers=self._servers)
+        await self._producer.start()
+        await self._consumer.start()
+
+    async def disconnect(self) -> None:
+        await self._producer.stop()
+        await self._consumer.stop()
+
+    async def subscribe(self, channel: str) -> None:
+        # Currently no support for pattern, in order to support both,
+        # 2 different consumers should be created,
+        # and the interface should be discussed.
+        # subscribe method is not cumulative, so we need to call it with all
+        # the channels.
+        self._consumer_channels.add(channel)
+        self._consumer.subscribe(topics=self._consumer_channels)
+
+    async def unsubscribe(self, channel: str) -> None:
+        # Unsubscribed is only possible with aiokafka consumer.
+        await self._consumer.unsubscribe()
+
+    async def publish(self, channel: str, message: typing.Any) -> None:
+        # Producer accepts only bytes, not sure how to deal with it
+        await self._producer.send_and_wait(channel, message.encode("utf8"))
```

Comment:
kafka accepts bytes only as message, any idea how to handle this? my current solution is quite basic, but it can be done better. I could just pass `message` as is, and let the aiokafka exception be triggered, so is up to the user to convert to bytes outside `broadcaster`

Reply:
This'll do right now. Actually we need to implement a serializers API, but let's treat that seperately.